crates/api_common/src/build_response.rs

```diff
@@ -187,15 +187,14 @@ pub async fn send_local_notifs(
 
       // Send an email to those local users that have notifications on
       if do_send_email {
-        let lang = &mention_user_view.local_user.interface_i18n_language();
-        let content = markdown_to_html(&comment_content_or_post_body);
-        send_email_to_user(
+        send_mention_email(
           &mention_user_view,
-          &lang.notification_mentioned_by_subject(&person.name),
-          &lang.notification_mentioned_by_body(&link, &content, &inbox_link, &person.name),
+          &comment_content_or_post_body,
+          person,
+          link,
           context.settings(),
         )
-        .await
+        .await;
```

Comment:
This will probably need an `.ok()`

Reply:
No because those functions dont return a result (same as before).